### PR TITLE
Feature/monitor

### PR DESCRIPTION
**Pull request by giulioungaretti**
TODO:
- [x] fix setup.py

usage

```python
from qcodes import ManualParameter
import qcodes as qc

y = ManualParameter("yolo")

from qcodes.tests.instrument_mocks import DummyInstrument, MockParabola

dac = DummyInstrument(name="dac")
dac1 = DummyInstrument(name="da2c")
par = MockParabola("parabola")

params = list(dac.parameters.values()) + \
         list(dac1.parameters.values()) + \
          list(par.parameters.values())  + [y]

monitor  = qc.Monitor(dac1.dac2, dac.dac1) 

# or
# monitor  = qc.Monitor(*params) 


# shows automatically but one can reopne the window if closed
monitor.show()

# monitor can be restarted in place

monitor  = qc.Monitor(dac1.dac2, dac.dac1) 
```

### CODE REVIEW

**Comment by giulioungaretti** [comment]
@jenshnielsen @WilliamHPNielsen  yo, check and merge if needed
I visiting the 👨‍⚕️today as well, may be back later on !

File: setup.py [hunk 1]
```diff
@@ -53,8 +53,8 @@ def readme():
       # if we want to install without tests:
       # packages=find_packages(exclude=["*.tests", "tests"]),
       packages=find_packages(),
-      package_data={'qcodes': ['widgets/*.js', 'widgets/*.css', 'config/*.json']},
-      install_requires= [
+      package_data={'qcodes': ['monitor/dist/*', 'config/*.json']},
+      install_requires=[
```
Comment (jenshnielsen) [inline setup.py:57]: This needs a dependency on websockets

**Comment by jenshnielsen** [comment]
When I try to run it I dont see any output show up but get the less helpfull error 

```
0:30: execution error: An error of type -10814 has occurred. (-10814)
```

**Comment by jenshnielsen** [comment]
If I manually open the webpage at port 3000 everything seems to be fine

File: setup.py (hunk 1 above)
Comment (giulioungaretti) [reply setup.py:57]: @jenshnielsen  yes! Good point!

**Comment by giulioungaretti** [comment]
@jenshnielsen  where do you get the error ?

**Comment by jenshnielsen** [comment]
Sorry I did do a bit more investigation that error is thrown when calling `webbrowser.open` It also happens when doing it manually on my mac, did not test on windows. Might be a config specific to the mac?

**Comment by jenshnielsen** [comment]
Is this ready to land

**Pull request merged**